### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by RoboNET**
Fix bug with cyrilic path and add VS 2015 to extension manifest

### CODE REVIEW

File: GradleBindings.VisualStudio/source.extension.vsixmanifest [hunk 1]
```diff
@@ -12,7 +12,13 @@
         <Tags>xamarin, gradle, maven, jcenter</Tags>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[12.0]" />
```
Comment (EgorBo) [inline GradleBindings.VisualStudio/source.extension.vsixmanifest]: @RoboNET hey, thanks for the pull-request, but doesn't it brake the VS2013 support?

Comment (MKorolev-igs) [inline GradleBindings.VisualStudio/source.extension.vsixmanifest]: I uninstall VS2013 and can`t test it. Could you do it?

Comment (RoboNET) [inline GradleBindings.VisualStudio/source.extension.vsixmanifest]: Sorry. It`s my second account. I forgot to logout )

**Comment by EgorBo** [comment]
I'll merge and test then :smile: thanks!

**Pull request merged**